TESTING.MD

```diff
@@ -1,22 +1,16 @@
-Cookbook TESTING doc
-====================
+# Cookbook TESTING doc
 
-Testing Prerequisites
----------------------
-Chef cookbooks require either a working ChefDK installation set as your system's default ruby or Ruby 2.0+ with bundler installed. Using ChefDK provides a consistent Ruby install, and is the suggested method. ChefDK can be downloaded at https://downloads.chef.io/chef-dk/
+## Testing Prerequisites
+
+Chef cookbooks require either a working ChefDK installation set as your system's default ruby or Ruby 2.0+ with bundler installed. Using ChefDK provides a consistent Ruby install, and is the suggested method. ChefDK can be downloaded at <<https://downloads.chef.io/chef-dk/>
 
 To ensure all gems are updated to their latest releases run `bundle install; bundle update` before running any Rake testing tasks.
```

Comment:
This stuff wasn't really updated by this PR and mostly needs to get rewritten.  It's some legacy stuff with modifications.